src/constants/ActionTypes.js

```diff
@@ -31,6 +31,9 @@ export const FINISH_ANSWERING_QUESTION = makeActionStrings(
 )
 export const DELETE_QUESTION = makeActionStrings('DELETE_QUESTION')
 
+// Delete All Questions
```

Comment:
Comment is probably unnecessary, it doesn't say anything new that isn't clear from the variable name.